### ISSUE TITLE
Shell Launcher V2 With Win UI 3 App

### ISSUE THREAD

**Issue by scout208** [Microsoft/Windows-iotcore-samples#488]
I'm trying to setup Shell Launcher with a Packaged Win UI 3 app as the shell. I have used Shell Launcher V2 with a UWP app in the past and it has worked fine, but I'm not sure how to set it up to use a Win UI 3 app that is packaged since it has an MSIX file, but it is technically not a UWP app. This is the XML I am trying to use and it looks like the app does start but it crashes immediately.

````
<?xml version="1.0" encoding="utf-8"?>
<ShellLauncherConfiguration xmlns="http://schemas.microsoft.com/ShellLauncher/2018/Configuration" xmlns:v2="http://schemas.microsoft.com/ShellLauncher/2019/Configuration">
  <Profiles>
    <DefaultProfile>
      <Shell Shell="%systemroot%\explorer.exe">
        <DefaultAction Action="RestartShell"/>
      </Shell>
    </DefaultProfile>
    <Profile Id="{814B6409-8C51-4EE2-95F8-DB39B70F5F68}">
      <Shell Shell="c4fa5ec0-d393-44e0-b6a9-87e8ddca3a2f_2xxtabv2m5ha2!App" v2:AppType="UWP">
        <ReturnCodeActions>
          <ReturnCodeAction ReturnCode="1" Action="DoNothing" />
        </ReturnCodeActions>
        <DefaultAction Action="RestartDevice"/>
      </Shell>
    </Profile>
  </Profiles>
  <Configs>
    <Config>
      <AutoLogonAccount />
      <Profile Id="{814B6409-8C51-4EE2-95F8-DB39B70F5F68}"/>
    </Config>
  </Configs>
</ShellLauncherConfiguration>
````

Has anyone had any experience setting this up?

**Comment by JensNordenbro** [comment]
@scout208 , did you solve this? I have a similar issue

**Comment by scout208** [comment]
@JensNordenbro Yes, I ended up figuring it out. If I remember correctly the issue was not with my XML at all but with some code I was using in my app to try and make the app run full screen. I removed that code and changed `<Shell Shell="c4fa5ec0-d393-44e0-b6a9-87e8ddca3a2f_2xxtabv2m5ha2!App" v2:AppType="UWP">` to `<Shell Shell="c4fa5ec0-d393-44e0-b6a9-87e8ddca3a2f_2xxtabv2m5ha2!App" v2:AppType="UWP" v2:AllAppsFullScreen="true">` and that ended up working out perfectly for me.
